File: challenges/3-area-trapezio/python/Apenas-Kev/trapezio.py
while True:
    try:
        altura = float(input('Qual a altura?: '))
        base_maior = float(input('Qual a base maior?: '))
        base_menor = float(input('Qual a base menor?: '))
        break;
    except ValueError:
        print(f'\n{"-" * 8} ERRO {"-" * 8}\nInsira somente numeros.\n')

print(f'A área do trapézio é: {(base_maior + base_menor) * altura / 2}')

File: challenges/3-area-trapezio/c/vaipc/trapezio.c
#include <stdio.h>

int main () {

  float basemaior, basemenor, altura, area;

  printf("Digite o tamanho da menor base: ");
  scanf("%f", &basemenor);
  getchar();

  printf("Digite o tamanho da base maior: ");
  scanf("%f", &basemaior);
  getchar();

  printf("Digite a altura: ");
  scanf("%f", &altura);
  getchar();

  area=(basemaior + basemenor)*altura/2;

  printf("A área do trapézio é igual a: %.1f", area);

  return 0;

}